src/utils/order.js

```diff
@@ -6,7 +6,7 @@ export const orderBars = (dataArr, lastBar) => {
   );
 
   const sortedFiltered =
-    filtered[0].name == 'Other' ? filtered : filtered.reverse();
+    filtered[0].name === 'Other' ? filtered : filtered.reverse();
 
```

Comment:
thanks for fixing that :)

Reply:
No problemo - the linter was complaining 😛